EndlessClient/UIControls/BlinkingLabel.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Diagnostics;
```

Comment:
I don't see the value in these changes - is there something wrong with `BlinkingLabel`? Strongly prefer to revert this file in its entirety.

Usually a crash in `base.Update()` shows up in `EndlessGame` due to collections being modified. This is *usually* caused by trying to update the collection on a background thread (collection was modified, enumeration may not continue).

Reply:
![image](https://github.com/ethanmoffat/EndlessClient/assets/70440468/593baaf9-7772-4019-9016-fdece877ea5d)

I get this issue if i revert or I can do the below instead

![image](https://github.com/ethanmoffat/EndlessClient/assets/70440468/851e5553-d6a0-4b77-9fcb-077f8a73f7ed)

Reply:
Resolved with `isUiControl` parameter when creating character renderers (to be fixed more elegantly later)